### PR TITLE
Edits to developers index.md

### PR DESCRIPTION
**Pull request by bobadair**
Edits to developers/index.md

Signed-off-by: Bob Adair <bob.github@att.net>

### CODE REVIEW

File: developers/index.md [hunk 1]
```diff
@@ -5,38 +5,37 @@ title: Developer Guide
 
 # Overview
 
-A variety of different parts of openHAB can be extended, we go through all of them.
+A variety of different parts of openHAB can be extended. We will cover the most important ones here.
```
Comment (Confectrician) [inline developers/index.md:8]:
```suggestion
A variety of different parts of openHAB can be extended.
We will cover the most important ones here.
```

Please use one line per sentence.
The website build will take car of the correct presentation.

File: developers/index.md [hunk 2]
```diff
@@ -116,6 +115,6 @@ This script is specific for binding addons. Follow these steps to generate your
 
 1. Open Eclipse or your favorite editor and import your new binding project
 
-Now you can start developing your NEW binding. We're looking forward to your pull request!
+Now you can now start developing your NEW binding. We're looking forward to your pull request!
```
Comment (Confectrician) [inline developers/index.md:118]: Are both `now` intended here?
The first one looks weird after the change so far.

**Review by Confectrician — COMMENTED** [review]
Hi @bobadair 

Thanks again.
Some minor comments below.

File: developers/index.md (hunk 2 above)
Comment (bobadair) [reply developers/index.md:119]: Oops! I'll fix that in a minute.

**Review by Confectrician — APPROVED** [review]
Thanks. 👍

**Pull request merged**